libs/openFrameworks/gl/ofGLUtils.cpp

```diff
@@ -253,6 +255,8 @@ int ofGetGlTypeFromInternal(int glInternalFormat){
 
 		case GL_STENCIL_INDEX:
 			return GL_UNSIGNED_BYTE;
+		case GL_BGRA:
```

Comment:
are you sure that should be `GL_BGRA`?  i believe it should say GL_UNSIGNED_BYTE, in which case you could add it  above next to GL_RGB